src/jsx.d.ts

```diff
@@ -977,7 +977,7 @@ export namespace JSXInternal {
 		i: HTMLAttributes<HTMLElement>;
 		iframe: HTMLAttributes<HTMLIFrameElement>;
 		img: HTMLAttributes<HTMLImageElement>;
-		input: HTMLAttributes<HTMLInputElement> & { defaultValue?: string };
+		input: HTMLAttributes<HTMLInputElement> & { defaultValue?: string | SignalLike<string> };
```

Comment:
Looks like this could be dropped all together, as `defaultValue` is already included in `HTMLAttributes` here: https://github.com/preactjs/preact/blob/master/src/jsx.d.ts#L714

Reply:
```suggestion
		input: HTMLAttributes<HTMLInputElement>;
```